development/building/with-docker.md

```diff
@@ -6,8 +6,6 @@ Once you installed the prerequisites and cloned the repository, run the followin
 
 ### Step 1 : Build
 
-> _**NOTE:**_ If you are an macOS M1 User then please keep in mind below things
-
 > Make sure you have ARM supported java installed
```

Comment:
this one seems extra as well?

Reply:
For local development, having a JVM with ARM support is a good idea. I think that we can leave that

Reply:
Sure, but this
> Make sure you have ARM supported java installed
looks weird without a notice that it's only for ARM users

Reply:
Removed it with 9a8190ba326d2daaeaff24756fbfbdba3055514e